### ISSUE TITLE
A Dataset of Alt Texts from HCI Publications: Analyses and Uses Towards Producing More Descriptive Alt Texts of Data Visualizations in Scientific Papers

### ISSUE THREAD

**Issue by XiyueWang-Miraikan** [Wotipati/Blind_Accessibility_Papers#537]
### Links
- Paper: https://dl.acm.org/doi/10.1145/3517428.3544796

### Abstract
Figures in scientific publications contain important information and results, and alt text is needed for blind and low vision readers to engage with their content. We conduct a study to characterize the semantic content of alt text in HCI publications based on a framework introduced by Lundgard and Satyanarayan [30]. Our study focuses on alt text for graphs, charts, and plots extracted from HCI and accessibility publications; we focus on these communities due to the lack of alt text in papers published outside of these disciplines. We find that the capacity of author-written alt text to fulfill blind and low vision user needs is mixed; for example, only 50% of alt texts in our sample contain information about extrema or outliers, and only 31% contain information about major trends or comparisons conveyed by the graph. We release our collected dataset of author-written alt text, and outline possible ways that it can be used to develop tools and models to assist future authors in writing better alt text. Based on our findings, we also discuss recommendations that can be acted upon by publishers and authors to encourage inclusion of more types of semantic content in alt text.

<img width="601" alt="スクリーンショット 2022-12-06 18 59 02" src="https://user-images.githubusercontent.com/91875613/205879609-6862cdf0-025c-4507-bb5d-f716acb3a9b4.png">
<img width="694" alt="スクリーンショット 2022-12-06 18 59 29" src="https://user-images.githubusercontent.com/91875613/205879731-babdd242-4325-49e4-a574-fb049c72911b.png">
<img width="711" alt="スクリーンショット 2022-12-06 19 00 20" src="https://user-images.githubusercontent.com/91875613/205879922-8ef28026-4dd4-469c-b03c-0a18cc6842be.png">
<img width="684" alt="スクリーンショット 2022-12-06 19 00 48" src="https://user-images.githubusercontent.com/91875613/205880049-a1fa1158-2cc0-48aa-b3bc-d7b8d124fd27.png">
